travis/script.sh

```diff
@@ -200,6 +200,15 @@ echo "Running test ${TEST}"
       echo "Docker compose container status"
       docker-compose -f docker-compose.yml ps
       ;;
+    unittests)
+      echo "run unittests scripts"
+      # change user id withn Docker container to user id of travis user
+      sed -i -e "s/USER\ 1001/USER\ `id -u`/g" ./Dockerfile.django
+      cp ./dojo/settings/settings.dist.py ./dojo/settings/settings.py
+      source ./docker/setEnv.sh release
+      docker-compose build
```

Comment:
we could use the `build_containers` function like it's done in the other functions

Reply:
using build containers i won't be able to run *docker-compose up -d* so that all the services can run and communicate together in the background and browsers will be able to access the service in *headless mode* within the travis runner

Reply:
I won't also be able to fetch password that i cn now set in environment for the login pages...